### PR TITLE
refactor(core): various fixes for missing-injectable migration

### PR DESCRIPTION
**Pull request by devversion**
See individual commits.

### CODE REVIEW

File: packages/core/schematics/migrations/missing-injectable/transform.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {forwardRefResolver} from '@angular/compiler-cli/src/ngtsc/annotations/src/util';
```
Comment (alxhub) [inline packages/core/schematics/migrations/missing-injectable/transform.ts]: We really need to put this in a better place...

**Review by alxhub — APPROVED** [review]

**Comment by alxhub** [comment]
[Presubmit](http://test/OCL:276504988:BASE:276505037:1571935006387:8c73b005)

File: packages/core/schematics/migrations/missing-injectable/transform.ts (hunk 1 above)
Comment (devversion) [reply packages/core/schematics/migrations/missing-injectable/transform.ts]: Agreed. I can submit a JIRA ticket for this.

**Comment by AndrewKushnir** [comment]
FYI, g3 presubmit looks good.

**Pull request closed without merging**